sutrofm/static/js/search-view.js

```diff
@@ -14,7 +14,7 @@
 
     enqueueTrack: function(trackKey) {
       $.ajax({
-        'url': '/api/party/'+window.roomId+'/enqueue',
+        'url': '/api/party/'+window.roomId+'/queue/enqueue',
```

Comment:
actually this is api spec https://github.com/sutrofm/sutrofm/wiki/Backend-Re-Write